client/src/components/FoodSeeker/SearchResults/ResultsFilters/FilterPanel.js

```diff
@@ -175,50 +176,52 @@ export default function FilterPanel({ mealPantry }) {
           gap: (theme) => theme.spacing(2),
         }}
       >
-        {["Show All", "Open Now", "Customized"].map((text) => (
+        {["Show All", "Open Now"].map((text) => (
           <FormControlLabel
             key={text}
             value={text}
             control={<Radio name={text} sx={checkedStyle} />}
-            label={
-              text === "Customized" ? (
-                <Stack direction="row" sx={{ width: "100%" }} gap={2}>
-                  {["day", "time"].map((key) => (
-                    <Select
-                      disabled={openTime.radio !== "Customized"}
-                      labelId={`${key}-label`}
-                      id={`${key}-select`}
-                      key={`${key}-select`}
-                      name={key}
-                      value={openTime.radio === "Customized" && openTime[key]}
-                      onChange={(e) => handleOpenTimeChange(e)}
-                      displayEmpty
-                      renderValue={(selected) => {
-                        return openTime.radio !== "Customized" ||
-                          selected.length === 0 ? (
-                          <em>{OPEN_TIME_DROPDOWNS[key].placeholder}</em>
-                        ) : (
-                          selected
-                        );
-                      }}
-                      MenuProps={{
-                        sx: { maxHeight: 180, width: "fit-content" },
-                      }}
-                    >
-                      {OPEN_TIME_DROPDOWNS[key].labels.map((label) => (
-                        <MenuItem key={label} label={label} value={label}>
-                          {label}
-                        </MenuItem>
-                      ))}
-                    </Select>
-                  ))}
-                </Stack>
-              ) : (
-                text
-              )
-            }
+            label={text}
           />
         ))}
+        <FormControlLabel
+          key="Customized"
+          value="Customized"
+          control={<Radio name="Customized" sx={checkedStyle} />}
+          label={
+            <Stack direction="row" sx={{ width: "100%" }} gap={2}>
+              {["day", "time"].map((key) => {
+                return (
+                  <Select
+                    disabled={openTime.radio !== "Customized"}
+                    labelId={`${key}-label`}
+                    id={`${key}-select`}
+                    key={`${key}-select`}
+                    name={key}
+                    value={openTime[key]}
+                    onChange={(e) => handleOpenTimeChange(e)}
+                    displayEmpty
+                    renderValue={(selected) => {
+                      if (openTime.radio !== "Customized" || !selected) {
+                        return <em>{OPEN_TIME_DROPDOWNS[key].placeholder}</em>;
+                      }
+                      return undefined;
```

Comment:
need the original logic, or the placeholder will not display any value after day and time are selected.
```
return openTime.radio !== "Customized" ||
  selected.length === 0 ? (
  <em>{OPEN_TIME_DROPDOWNS[key].placeholder}</em>
) : (
  selected
);
```
Current branch behavior:

https://github.com/hackforla/food-oasis/assets/68657634/67ff2c7e-e874-40b5-bd7c-3c6017c9cbfb